plugins/autowarmth.koplugin/main.lua

```diff
@@ -160,18 +160,35 @@ end
 
 AutoWarmth._onEnterStandby = AutoWarmth._onSuspend
 
+function AutoWarmth:_onToggleNightMode()
+    if self.control_nightmode and not AutoWarmth.hide_nightmode_warning then
+        UIManager:show(ConfirmBox:new{
+            text = _("Night mode changed by the user.\nThe AutoWarmth plugin might change it again."),
+            ok_text = _("Show warning again"),
+            cancel_text = _("Hide warning"),
```

Comment:
Also "Disable AutoWarmth"?

Reply:
Oh, yeah. But I would just say something like "Disable AutoWarmth's toggling" (as the rest of AutoWarmth is not affected by the toggle). But that sound weird.

Reply:
Disable auto-toggle?

Reply:
Maybe "Disable AutoWarmth's control"?

Reply:
Perhaps instead of a button a checkmark along these lines. Then it doesn't matter that your suggestions are all very long. ;-)
<img src=https://user-images.githubusercontent.com/202757/199461484-fd4bc163-59a6-418d-8ba8-0762e4d555f3.png width=40%>

Reply:
I will go for that.

Reply:
This is how it would look like:
![grafik](https://user-images.githubusercontent.com/36999612/199486484-5829e6bf-9324-40fa-b921-881e2ee50ba9.png)

Reply:
Removing the periods might make it look a little lighter.

Seems fine to me otherwise.